src/main/java/uk/gov/indexer/dao/DestinationDBQueryDAO.java

```diff
@@ -0,0 +1,70 @@
+package uk.gov.indexer.dao;
+
+import org.codehaus.jackson.JsonNode;
+import org.codehaus.jackson.map.ObjectMapper;
+import org.postgresql.util.PGobject;
+import org.skife.jdbi.v2.TransactionIsolationLevel;
+import org.skife.jdbi.v2.sqlobject.Bind;
+import org.skife.jdbi.v2.sqlobject.SqlQuery;
+import org.skife.jdbi.v2.sqlobject.SqlUpdate;
+import org.skife.jdbi.v2.sqlobject.Transaction;
+
+import java.nio.charset.Charset;
+import java.sql.SQLException;
+import java.util.List;
+
+public abstract class DestinationDBQueryDAO implements DBQueryDAO {
```

Comment:
The word `Query` in the class name implies that this is a read-only DAO, when it also performs updates.  How about `DestinationDBUpdateDAO`?